docs/traffic-policy/actions/index.mdx

```diff
@@ -0,0 +1,171 @@
+    phases:
+      - on_http_request
+    categories:
+      - security
+
+  - type: compress-response
+    name: Compress Response
+    description: Compress HTTP response bodies from your upstream server.
+    phases:
+      - on_http_request
+      - on_http_response
+    categories:
+      - response-modification
+
+  - type: custom-response
+    name: Custom Response
+    description: Send a custom HTTP response directly back to the client.
+    phases:
+      - on_http_request
+      - on_http_response
+    categories:
+      - response-modification
+
+  - type: deny
+    name: Deny
+    description: Deny incoming traffic to your endpoint at the HTTP layer.
+    phases:
+      - on_http_request
+      - on_tcp_connect
+    categories:
+      - traffic-control
+      - connection-modification
+
+  - type: forward-internal
+    name: Forward Internal
+    description: Forward traffic to an internal endpoint within the same ngrok account.
+    phases:
+      - on_http_request
+      - on_tcp_connect
+    categories:
+      - traffic-control
+      - connection-modification
+
+  - type: jwt-validation
+    name: JWT Validation
+    description: Validate JSON Web Tokens (JWT) on your incoming requests.
+    phases:
+      - on_http_request
+    categories:
+      - security
+
+  - type: log
+    name: Log
+    description: Add log metadata to events for logging and monitoring.
+    phases:
+      - on_http_request
+      - on_http_response
+      - on_tcp_connect
+    categories:
+      - connection-modification
+      - response-modification
+      - request-modification
+
+  - type: rate-limit
+    name: Rate Limit
+    description: Rate limit incoming traffic to your endpoint before it hits your upstream servers.
+    phases:
+      - on_http_request
+    categories:
+      - traffic-control
+      - request-modification
+
+  - type: redirect
+    name: Redirect
+    description: Redirect users through URL transformations using regular expressions.
+    phases:
+      - on_http_request
+    categories:
+      - request-modification
+
+  - type: remove-headers
+    name: Remove Headers
+    description: Remove headers from incoming requests or outgoing responses.
+    phases:
+      - on_http_request
+      - on_http_response
+    categories:
+      - request-modification
+      - response-modification
+
+  - type: restrict-ips
+    name: Restrict IPs
+    description: Allow or deny incoming traffic based on the client IP.
+    phases:
+      - on_http_request
+      - on_tcp_connect
+    categories:
+      - security
+      - connection-modification
+
+  - type: url-rewrite
+    name: URL Rewrite
+    description: Rewrite request URLs transparently using regular expressions.
+    phases:
+      - on_http_request
+    categories:
+      - request-modification
+
+  - type: verify-webhook
+    name: Verify Webhook
+    description: Validate incoming signatures against a known secret to ensure authenticity.
```

Comment:
```suggestion
    description: Validate incoming webhook requests to ensure authenticity.
```

Reply:
Intentionally written verbose to try and keep each to 2 lines.